garrysmod/lua/vgui/dhtml.lua

```diff
@@ -1,4 +1,4 @@
-
+local color_purple = Color( 255, 160, 255 )

```

Comment:
Minor, but this should be moved closer to where it is used i.e. before `PANEL:ConsoleMessage`